Python/Product/PythonTools/PythonTools/Intellisense/ExtractMethodSuggestedActionSourceProvider.cs

```diff
@@ -82,7 +82,9 @@ class ExtractMethodSuggestedActionSource : ISuggestedActionsSource {
                 if (MethodExtractor.CanExtract(_view) == true) {
                     return new SuggestedActionSet[] {
                     new SuggestedActionSet(
+                        requestedActionCategories.First(),
```

Comment:
Unrelated change? Should safely handle `InvalidOperationException` if you're going to use `First()`.

Reply:
I am going to undo this one. I was not able to build because VS kept complaining that the version that is used was Obsolete.